paasta_tools/kubernetes_tools.py

```diff
@@ -1501,6 +1579,18 @@ def get_boto_secret_hash(self) -> str:
             namespace=self.get_namespace(),
         )
 
+    def get_crypto_secret_hash(self) -> str:
```

Comment:
i'm surprised mypy isn't failing on this - `get_kubernetes_secret_signature` returns `Optional[str]`, not `str`